lib/Service/ApiService.php

```diff
@@ -106,10 +106,17 @@ public function create($fileId = null, $filePath = null, $token = null, $guestNa
 		}
 
 		$session = $this->sessionService->initSession($document->getId(), $guestName);
+		try {
+			$baseFile = $this->documentService->getBaseFile($document->getId());
+			$content = $baseFile->getContent();
+		} catch (NotFoundException $e) {
+			$content = null;
+		}
```

Comment:
This really should not happen as the file was just created. We should probably log it if we end up here.